Index: cmake-build-debug/.ninja_log
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># ninja log v6\n5\t72\t1732037867912174615\tCMakeFiles/calculator.dir/main.cpp.o\tb54b69d22b3495e5\n1\t115\t1732037892674468107\tCMakeFiles/calculator.dir/stack.cpp.o\t3f67dc7f1432a99\n1\t19\t1732038937209727396\tCMakeFiles/calculator.dir/main.cpp.o\tb54b69d22b3495e5\n2\t428\t1732038937210727240\tCMakeFiles/calculator.dir/stack.cpp.o\t3f67dc7f1432a99\n
===================================================================
diff --git a/cmake-build-debug/.ninja_log b/cmake-build-debug/.ninja_log
--- a/cmake-build-debug/.ninja_log	(revision df5ea0957ade420af59830b634c6097802b57e18)
+++ b/cmake-build-debug/.ninja_log	(date 1732039242661)
@@ -3,3 +3,7 @@
 1	115	1732037892674468107	CMakeFiles/calculator.dir/stack.cpp.o	3f67dc7f1432a99
 1	19	1732038937209727396	CMakeFiles/calculator.dir/main.cpp.o	b54b69d22b3495e5
 2	428	1732038937210727240	CMakeFiles/calculator.dir/stack.cpp.o	3f67dc7f1432a99
+1	424	1732039152602778680	CMakeFiles/calculator.dir/stack.cpp.o	3f67dc7f1432a99
+2	435	1732039181509919405	CMakeFiles/calculator.dir/stack.cpp.o	3f67dc7f1432a99
+2	444	1732039233309388916	CMakeFiles/calculator.dir/stack.cpp.o	3f67dc7f1432a99
+2	428	1732039242235223865	CMakeFiles/calculator.dir/stack.cpp.o	3f67dc7f1432a99
